Documentation/ceph-cluster-crd.md

```diff
@@ -298,6 +299,8 @@ Here are the current minimum amounts of memory in MB to apply so that Rook will
 - `mds`: 4096MB
 - `rbdmirror`: 512MB
 
+Rook does not enforce any minimum limit nor request on the prepare OSD pod. We have seen that pod taking up to 50 MB, but depending on the OSD scenario, that value might need to be higher. So setting a value to at least 50 MB should be a good start, but 100 MB might be safer.
```

Comment:
Nit: it might be good to specifically note that this is the `prepareosd` key for easier cross referencing.

Reply:
oops, before i merged I was thinking it already mentioned the prepare osd pod, but you were suggesting to mention the key name.